src/com/intellij/idea/plugin/hybris/tools/remote/console/impl/HybrisGroovyConsole.kt

```diff
@@ -1,6 +1,6 @@
 /*
- * This file is part of "hybris integration" plugin for Intellij IDEA.
- * Copyright (C) 2014-2016 Alexander Bartash <AlexanderBartash@gmail.com>
+ * This file is part of "SAP Commerce Developers Toolset" plugin for Intellij IDEA.
+ * Copyright (C) 2023 EPAM Systems <hybrisideaplugin@epam.com>
```

Comment:
@mlytvyn Hi. This is the first and last friendly reminder not to change copyright on the existing code, please stop doing it.

Reply:
@AlexanderBartash , thank you for noticing, let's work together and adjust the Copyright content, do you have any suggestions?

Reply:
Simply please do not change it. Have respect for other people's work. We have spent several years of our time and dozens of thousands of dollars worth of our own money to create this product. You can not re-copyright it simply because you've changed a couple of lines.

Reply:
@AlexanderBartash , still, some part of code is completely new and is not written/contributed by any initial plugin developers, in that case initial copyright cannot be used.

Instead, we can have two copyright headers:
1. one for initial code (2014-2016 Alexander, 2023+ EPAM)
2. and second for new code (2023+ EPAM)

Reply:
@mlytvyn Two copyrights is ok. Overwriting to EPAM is not.

If it is difficult for you to track which file was modified by whom you can add two copyright lines to all of them, but then all new files will also be copyrighted to me and epam. I do not need any ownership on the code I did not write, but if it makes you life easier, then fine.

Reply:
@AlexanderBartash , what do you think of Java -> Kotlin and huge changes to the code?

Reply:
If you spent 5 years developing the plugin at your free time and payed dozens of thousands of dollars of you worn money to other developers to help you support it. Then someone would come, convert your java code to Kotlin and call it theirs and remove all mentions of you, would you be ok with that?

Reply:
@AlexanderBartash , take into account time and efforts of the person who is not just converting the code, but changing the logic, usually what is left is name of the class. In such a case it may not be fair to say that initial devs put all efforts.
Also, if we're talking about "initial idea" conversion from language to language usually results into a new code and new  owners, that's like some algorithm/logic in javascript being converted into java under new copyright/authors.

Reply:
@mlytvyn I do not want to get into an argument with you what constitutes an ownership and what is not. I asked you not to change copyright on the existing code even if you refactor it. You will not get permission from me to do that. I want to be very explicit about it: I am not allowing to re-copyrighit it.

Also to be fair.
> new owners
You do not re-copyright it to some "new owners" or code authors or even yourself. You re-copyright it to a corporation.

Reply:
@AlexanderBartash , I was able to adjust copyright template in a way it will automatically detect initial copyright of yours. Let's hope that it will zero-down misunderstanding and ensure that any new contributor will not be questioned by any means.